src/cascadia/Remoting/Peasant.cpp

```diff
@@ -31,6 +31,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     {
         _id = id;
     }
+

```

Comment:
Fix comment and possibly the variable name `testPID` now that you made the second constructor public.

Reply:
FYI: the same goes for the Monarch too